.github/workflows/builds.yml

```diff
@@ -56,6 +56,9 @@ jobs:
           rsync -a --exclude .git --delete . ~/lxd-pkg-snap-lp/
           cd ~/lxd-pkg-snap-lp
           lxd-snapcraft -set-version "${ver[0]}" -set-source-commit "${ver[1]}"
+          # Test build lxd-migrate
```

Comment:
Can you add this as a new step between `Install Go` and `Trigger Launchpad snap build` called `Test lxd-migrate build`